tests/tools/jira/test_jira_api.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from unittest.mock import Mock
+from tools.jira_client import JiraAPI, CLOSED_STATUS
+
+
+def get_mock_issue(key="foobar"):
+    issue = Mock()
+    issue.key = key
+    return issue
```

Comment:
```suggestion
    return Mock(key=key)
```
maybe you don't even need this function as it's much shorter with its body